R/get_stan_data.R

```diff
@@ -476,7 +476,7 @@ get_ww_values <- function(ww_data,
 
     # Get the vector of log wastewater concentrations
     log_conc <- ww_data |>
-      dplyr::mutate(log_conc = as.numeric(log(!!sym(
+      dplyr::mutate(log_conc = as.numeric(log(!!rlang::sym(
         ww_measurement_col_name
       ) + 1e-8))) |>
```

Comment:
I'd use the `.data` pronoun here. NB [some R CMD check details](https://rlang.r-lib.org/reference/dot-data.html#where-does-data-live-)

```suggestion
      dplyr::mutate(log_conc = as.numeric(log(.data[[ww_measurement_col_name]] + 1e-8))) |>
```

Also, why is the `as.numeric()` call needed?

Reply:
Would be good to make the small added on value (here `1e-8`) configurable, but that can be a separate PR.

Reply:
Hmm the `as.numeric()` was probably a holdover from a previous syntax that wasnt working properly. This makes sense.